homeassistant/components/matter/discovery.py

```diff
@@ -2,10 +2,9 @@
 
 from __future__ import annotations
 
-from collections.abc import Generator
-
 from chip.clusters.Objects import ClusterAttributeDescriptor
 from matter_server.client.models.node import MatterEndpoint
+from typing_extensions import Generator
```

Comment:
Can this change be reverted?
`Generator` should be imported from collections.abc.

Reply:
OK, I think that slipped through in the original PR. Lets address that in a follow up PR.

Reply:
@epenet https://github.com/home-assistant/core/pull/121236